docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.8'
+
+services:
+  hypha:
+    build: .
+    image: amun-ai/hypha
+    container_name: hypha
+    command: --port 9520 --host=0.0.0.0
```

Comment:
should this be more like the `CMD` in the `Dockerfile`

Reply:
I think you are right, we have it already https://github.com/amun-ai/hypha/blob/81ced64019c72335ee6c22d808cee91b4304ca8d/Dockerfile#L79

Reply:
Yeah the entry point in the dockerfile has it already. This PR was going to add in the permissions fix as well, but I believe @oeway fixed that already during the time this PR has been open. I've rebased on the main branch and now this PR is mostly trying to put some container testing in place. 

I'm now just checking to see if bumping the hypha version downstream fixes the permissions issues.

P.s
I think the default port on the original server this ran on was 9000 and that then became the port for the k8s cluster etc etc.

Reply:
Hi @ctr26 I think the latest one should work for non-privileged  rancher cluster, we have tested it with the helm chart included in the repo: https://github.com/amun-ai/hypha/tree/main/helm-charts

So maybe we can close this PR?

Reply:
Yep, I saw that you tried to fix the permission. I'm now updating the bioengine image with the newest hypha to make the egi stuff work. 

I think it's a good idea for the docker image testing that I've put here to still be added as it'll be useful for the future.

But otherwise this PR is now rebased to use your dockerfile